src/components/AllArticles.jsx

```diff
@@ -1,36 +1,43 @@
 import { useEffect, useState } from "react";
+import { fetchAllArticles } from "../api";
+
 
 function AllArticles() {
-    const [articles, setArticles] = useState([]);
+
+    const [isLoading, setLoading] = useState(true);
+    const [list, setList] = useState([]);
 
     useEffect(() => {
-        fetch(`https://nc-news-project5.onrender.com/api/articles`)
-            .then((response) => {
-                return response.json();
-            })
-            .then((data) => {
-                setArticles(data.articles);
-            });
+        fetchAllArticles().then((articles) => {
+            setList(articles);
+            setLoading(false);
+        })
     }, []);
 
     return (
-        <div>
-            <h2>Articles</h2>
-            {articles.map((article) => {
-                return (
-                    <div key={article.article_id}>
-                        <p>Title: {article.title}</p>
-                        <p>Topic: {article.topic}</p>
-                        <p>Author: {article.author}</p>
-                        <img src={article.article_img_url}></img>
-                        <p>Body: {article.body}</p>
-                        <p>Comments: {article.comments}</p>
-                        <p>Votes: {article.votes}</p>
-                        <p>Created at: {article.created_at}</p>
-                    </div>
-                );
-            })}
-        </div>
+        <section id="articles">
+            <h2>All Articles</h2>
+            {isLoading ? (
+                <h3>Loading...</h3>
+            ) : (
+                <ul>
+                    {list.map((article) => {
+                        return (
+                            <li key={article.article_id}>
```

Comment:
I'd think about seperating this out into a seperate component ArticleCard for example.